src/app/app-state.ts

```diff
@@ -93,7 +97,11 @@ export class AppState {
             map(({ data }) => data),
             shareReplay(1),
         )
-
+        this.connectedLocal$ = (
+            this.environmentClient.webSocket.ws['_log'] as unknown as {
```

Comment:
I missed it, but we should not use the '_log' private attribute.
We can for instance expose in 'WebSocket' an attribute 'connected$' (a combine latest for the `ws['_log'].connected$` & 'ws['_data'].connected$`. This is in the library `@youwol/local-youwol-client`).